### PR TITLE
Michaels edits

### PR DESCRIPTION
**Pull request by roeschter**

### CODE REVIEW

File: nats-concepts/jetstream/readme.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ JetStream is built-in to `nats-server` and you only need 1 (or 3 or 5 if you wan
...
 JetStream was created to solve the problems identified with streaming in technology today - complexity, fragility, and a lack of scalability. Some technologies address these better than others, but no current streaming technology is truly multi-tenant, horizontally scalable, and supports multiple deployment models. No other technology that we are aware of can scale from edge to cloud under the same security context while having complete deployment observability for operations.
 
+### Additional capabilities enabled by Jetstream
+The JetStream persistency layer enables additional use cases typically not found in messaging systems. Being build on top of JetStream they inherit the core capabilities of JetStream, replication, security, routing limits and mirroring.  
+
+* [Key Value Store](#key-value-store) A map (associative array) with atomic operations
+* [Object Store](#object-store) A chunked file store
+
+
+Such capabilities are commonly the domain of in memory databases. It is the goal of NATS and JetStream to provide the developer
```
Comment (gcolliso) [inline nats-concepts/jetstream/readme.md]: @roeschter looks like this sentence didn't get finished? Something is missing.

Comment (roeschter) [reply nats-concepts/jetstream/readme.md]: Commited and edit

**Comment by gcolliso** [comment]
@roeschter I will review the latest updates & merge here but this should be it for this PR. I don't like them getting too long as it can cause conflicts should another update come in for the files in this PR.

**Comment by roeschter** [comment]
Totally, please blame it on lack of experience with GIT. I didn't create another fork in time.

**Pull request merged**